docker_test.go

```diff
@@ -46,6 +52,58 @@ import (
 	"github.com/tidwall/gjson"
 )
 
+type PostgresTestT struct {
```

Comment:
Lets follow the new convention and remove `T` from all the structs.

```suggestion
type PostgresTest struct {
```

Reply:
Removed T suffix from all structs.

Reply:
Done.